rest/connector-generator/src/main/java/io/syndesis/connector/generator/swagger/SwaggerHelper.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.connector.generator.swagger;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import io.swagger.models.Swagger;
+import io.syndesis.core.Json;
+
+public final class SwaggerHelper {
+
+    private SwaggerHelper() {
+        // utility class
+    }
+
+    /* default */ static String serialize(final Swagger swagger) {
```

Comment:
Is this helper class really needed ? The method is only called once and hence could be inlined, too.

Reply:
I refactored it out seemed at the point that I might be used more. I was getting (correct) complaints from PMD about SwaggerConnectorGenerator being a god class...